app/models/lock.rb

```diff
@@ -19,13 +21,23 @@ def global?
   end
 
   def summary
-    "Locked by #{user.try(:name) || 'Unknown user'} #{time_ago_in_words(created_at)} ago"
+    unlock = " and will unlock in #{time_ago_in_words(delete_at)}" if delete_at
+
+    "Locked by #{user.try(:name) || 'Unknown user'} #{time_ago_in_words(created_at)} ago#{unlock}"
   end
 
   def reason
     description.blank? ? "Description not given." : description
   end
 
+  def self.remove_expired_locks
+    Lock.where("delete_at IS NOT NULL and delete_at < CURRENT_TIMESTAMP").find_each(&:soft_delete)
```

Comment:
Might as well soft_delete! since you're checking exceptions but not the result of this method